build-linux/rpm/postinst

```diff
@@ -40,11 +40,7 @@ install_init () {
 install_systemd () {
     cp /var/lib/pdagent/scripts/pdagent.service /lib/systemd/system
     systemctl enable pdagent
-    if [ -z $DOCKER_DEV_ENV ]; then
-        # Prevent starting pdagent in Docker environment because systemd 
-        # will do that after running the Docker systemd process
-        systemctl start pdagent
-    fi
```

Comment:
This section isn't required anymore since `pdagent` is installed after the container has been spun up, more like on a real system.